zello-sdk-sample/src/main/java/com/zello/sdk/sample/TalkActivity.java

```diff
@@ -250,11 +250,11 @@ public void onClick(View v) {
 
 		_dirtyContacts = true;
 
-		// Use to connect to an app installed from an apk obtained from https://www.zellowork.com
-		//Zello.getInstance().configure("net.loudtalks", this);
-
-		// Use with an app installed from a generic PTT SDK apk obtained from https://github.com/zelloptt/zello-android-client-sdk/releases
-		Zello.getInstance().configure("com.pttsdk", this);
+		// Pass net.loudtalks to connect to an app installed from an apk obtained from https://www.zellowork.com
+		// Pass com.pttsdk to connect to a PTT SDK app obtained from https://github.com/zelloptt/zello-android-client-sdk/releases
+		// Pass com.loudtalks to connect to Zello app installed from the Google Play Store
+		// Or, pass null to automatically choose the app to connect to in the following order of preference: com.loudtalks, net.loudtalks, com.pttsdk
+		Zello.getInstance().configure(null, this);
```

Comment:
Relatively minor, but if we are expecting people to pass "null" here for the default behavior, it would be nice to overload the `Zello.configure` method to make a single-parameter version

Reply:
Good point. Adding.